arch/xtensa/src/esp32/esp32_wireless.h

```diff
@@ -0,0 +1,106 @@
+/****************************************************************************
+ * arch/xtensa/src/esp32/esp32_wireless.h
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __ARCH_XTENSA_SRC_ESP32_ESP32_WIRELESS_H
+#define __ARCH_XTENSA_SRC_ESP32_ESP32_WIRELESS_H
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <nuttx/config.h>
+
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+
+#define CONFIG_ESP32_SUPPORT_MULTIPLE_PHY_INIT_DATA_BIN 0
```

Comment:
```suggestion
#define CONFIG_ESP_PHY_MULTIPLE_INIT_DATA_BIN 0
```
IDF marks `CONFIG_ESP32_SUPPORT_MULTIPLE_PHY_INIT_DATA_BIN` as deprecated option, so it is better to use the updated config for making the driver future-proof.

Reply:
Unfortunately the current IDF header files are old and doesn't have this new definition. Let keep it for now.